nautobot/core/api/filter_backends.py

```diff
@@ -22,6 +22,7 @@ def get_filterset_kwargs(self, request, queryset, view):
             "include",  # used to include computed fields (excluded by default)
             "limit",  # pagination
             "offset",  # pagination
+            "ordering",  # ordering of results
```

Comment:
Looks like the name of this param is configurable in settings.py - should we call it `sort` instead of `ordering` as the 1.x UI uses `sort`?

Reply:
Or do we follow what the ORM calls it: `order_by`?

Reply:
☝️ Please see PR description